docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -6,210 +6,229 @@ description: Secretless Broker Documentation
-
-**5.** To ensure the **PostgreSQL StatefulSet** pod has started and is healthy, run the command:
-```bash
-kubectl --namespace quick-start-backend-ns get pods
-```
-<pre>
-NAME      READY     STATUS    RESTARTS   AGE
-pg-0      1/1       Running   0          6s
-</pre>
+      serviceName: quick-start-backend
+      selector:
+        matchLabels:
+          app: quick-start-backend
+      template:
+        metadata:
+          labels:
+            app: quick-start-backend
+        spec:
+          securityContext:
+            fsGroup: 999
+          containers:
+          - name: quick-start-backend
+            image: postgres:9.6
+            imagePullPolicy: IfNotPresent
+            ports:
+              - containerPort: 5432
+            env:
+              - name: POSTGRES_DB
+                value: postgres
+              - name: POSTGRES_USER
+                value: security_admin_user
+              - name: POSTGRES_PASSWORD
+                value: security_admin_password
+            volumeMounts:
+            - name: backend-certs
+              mountPath: "/etc/certs/"
+              readOnly: true
+            args: ["-c", "ssl=on", "-c", "ssl_cert_file=/etc/certs/server.crt", "-c", "ssl_key_file=/etc/certs/server.key"]
+          volumes:
+          - name: backend-certs
+            secret:
+              secretName: quick-start-backend-certs
+              defaultMode: 384
+    EOF
+    ```
+    **Note 1:** _In the manifest above, the certificate files for your database server are
+    mounted in a volume with `defaultMode: 384`, giving it permissions `0600`. (Why?
+     Because `600` in base 8 = `384` in base 10)._
+    
+    **Note 2:** _The pod is deployed with `999` as the group associated with
+    any mounted volumes, as indicated by `fsGroup: 999`.  `999` is a the static postgres
+    gid, defined in the postgres Docker image._
+
+1. Deploy the **PostgreSQL StatefulSet**:
+    ```bash
+    kubectl --namespace quick-start-backend-ns apply -f pg.yml
+    ```
+    <pre>
+    statefulset "pg" created
+    </pre>
+
+    This StatefulSet uses the DockerHub
+    [**postgres:9.6**](https://hub.docker.com/r/library/postgres/) container.
+
+    On startup, the container creates a superuser from the environment
+    variables `POSTGRES_USER` and `POSTGRES_PASSWORD`, which 
+    we set to the values `security_admin_user` and `security_admin_password`,
+    respectively.
+
+    Going forward, we'll call these values the **admin-credentials**, to distinguish
+    them from the **application-credentials** our application will use.
+
+    In the scripts below, we'll refer to the admin-credentials by the
+    environment variables `SECURITY_ADMIN_USER` and `SECURITY_ADMIN_PASSWORD`.
+
+1. To ensure the **PostgreSQL StatefulSet** pod has started and is healthy, run
```

Comment:
it took a minute or so for this to report healthy for me, not sure whether that's worth noting here